package-lock.json

```diff
@@ -21898,11 +21898,6 @@
 			"resolved": "https://registry.npmjs.org/postcss-value-parser/-/postcss-value-parser-4.1.0.tgz",
 			"integrity": "sha512-97DXOFbQJhk71ne5/Mt6cOu6yxsSfM0QGQyl0L25Gca4yGWEGJaig7l7gbCX623VqTBNGLRLaVUCnNkcedlRSQ=="
 		},
-		"preact": {
-			"version": "10.5.7",
```

Comment:
No idea why it was removed here - wasn't this a prior PR?

Reply:
Yes; however, I am not concerned about this.